tests/test_testPipeline.py

```diff
@@ -39,7 +39,10 @@
     MockCalibrateTask, MockGetTemplateTask, \
     MockAlardLuptonSubtractTask, MockDetectAndMeasureTask, MockTransformDiaSourceCatalogTask, \
     MockRBTransiNetTask, MockDiaPipelineTask, MockFilterDiaSourceCatalogTask, \
-    MockSpatiallySampledMetricsTask
+    MockSpatiallySampledMetricsTask, MockLoadDiaCatalogsTask
+from lsst.daf.butler import Timespan
+from lsst.pipe.base.utils import RegionTimeInfo
+from lsst.sphgeom import UnitVector3d
```

Comment:
I recommend sticking to the established (dependency) order, so that it's easier for future developers to see that the import already exists.